test/e2e/framework/network/utils.go

```diff
@@ -186,23 +211,29 @@ func (config *NetworkingTestConfig) EndpointHostnames() sets.String {
 //   at least once.
 // - maxTries is the maximum number of curl attempts. If this many attempts pass
 //   and we don't see all expected endpoints, the test fails.
-// - expectedEps is the set of endpointnames to wait for. Typically this is also
-//   the hostname reported by each pod in the service through /hostName.
+// - targetIP is the source Pod IP that will dial the given dialCommand using the given protocol.
+// - dialCommand is the command that the targetIP will send to the targetIP using the given protocol.
+//   the dialCommand should be formatted properly for the protocol (http: URL path+parameters,
+//   udp: command%20parameters, where parameters are optional)
+// - expectedResponses is the set of responses to wait for. The responses are based on
```

Comment:
What does this mean? are expectedResponses compared string wise with the result , etc.
As the caller of the func, reading this I would not know how to use this function...

Reply:
Amended a small change, although I don't know how to better explain it than through the example below.